### PR TITLE
Added Boarding event; fixed TimelineEventIcon to display Icon or Text…

### PR DESCRIPTION
**Pull request by RobinCsl**
…Icon

Note: There probably will be some conflicts with my previous PR as I had to modify some of the same files. I will fix them after the first PR is landed.

### CODE REVIEW

File: app/MMB/src/scenes/timeline/TimelineEventIcon.js [hunk 1]
```diff
@@ -2,25 +2,27 @@
...
 import * as React from 'react';
 import { View } from 'react-native';
-import { StyleSheet, Color, Icon } from '@kiwicom/mobile-shared';
+import { StyleSheet, Color, Icon, TextIcon } from '@kiwicom/mobile-shared';
 
 const size = 17;
 
 export type Props = {|
-  +name: string,
+  +type: 'Icon' | 'TextIcon',
```
Comment (tbergq) [inline app/MMB/src/scenes/timeline/TimelineEventIcon.js]: I think we should rather pass in the Icon component here.

Comment (RobinCsl) [reply app/MMB/src/scenes/timeline/TimelineEventIcon.js]: Ok, I will refactor this.

File: app/MMB/src/scenes/timeline/events/BoardingTimelineEvent.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  BoardingTimelineEvent,
+  graphql`
+    fragment BoardingTimelineEvent on BoardingTimelineEvent {
+      timestamp
+    }
+  `,
+);
+
+const styles = StyleSheet.create({
+  note: { fontSize: 13, color: Color.ink.light },
```
Comment (tbergq) [inline app/MMB/src/scenes/timeline/events/BoardingTimelineEvent.js]: I should probably be an eslint-rule, that is much more readable

```
note: { 
fontSize: 13, 
color: Color.ink.light,
},
```

Comment (RobinCsl) [reply app/MMB/src/scenes/timeline/events/BoardingTimelineEvent.js]: We can add 
```
"object-property-newline": ERROR
```
to `.eslintrc.js` to fix this. I agree with you.

Comment (tbergq) [reply app/MMB/src/scenes/timeline/events/BoardingTimelineEvent.js]: Yes, please do so

Comment (RobinCsl) [reply app/MMB/src/scenes/timeline/events/BoardingTimelineEvent.js]: I'll make a different PR for this, it is modifying too many files after I add this rule.

Comment (tbergq) [reply app/MMB/src/scenes/timeline/events/BoardingTimelineEvent.js]: Right, ok, then I would say leave it for now. Not that important. 

Some times it makes sense to write it on one line I guess.

**Comment by RobinCsl** [comment]
I have refactored the way to pass icons. 

**Note**: This PR needs the previous one to be merged before, and the resulting conflicts to be fixed, before it can be merged.

File: app/MMB/src/scenes/timeline/events/BoardingTimelineEvent.js (hunk 2 above)
Comment (RobinCsl) [reply app/MMB/src/scenes/timeline/events/BoardingTimelineEvent.js]: The notes are taken care of in the other PR https://github.com/kiwicom/mobile/pull/762

**Pull request merged**